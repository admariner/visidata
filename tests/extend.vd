sheet	col	row	longname	input	keystrokes	comment
			open-file	tests/data2.tsv	o	
			open-file	tests/data1.tsv	o	
data2	Key		key-col		!	
data1	Key		key-col		!	
data1			sheets-stack		S	
sheets		キdata2	select-row		s	
sheets		キdata1	select-row		s	
sheets			join-selected	extend	&	
data1+data2	A		freq-col		F	
data1			dup-rows		g"	
data1_copy	A		key-col		!	
data1_copy	Key		key-col		!	
data1_copy			sheets-all		S	
sheets_all		キdata1_copy	select-row		s	
sheets_all		キdata1+data2_A_freq	select-row		s	
sheets_all			join-selected	extend	&	
data1+data2_A_freq+data1_copy	histogram		hide-col		-	Hide current column
