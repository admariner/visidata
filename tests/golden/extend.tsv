A	count	percent	data1_copy_Key	data1_copy_B
a1	1	33.33	1	b1
c1	1	33.33	2	d1
e1	1	33.33	2	f1
